clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -840,13 +929,21 @@ raw2trace_t::process_header(raw2trace_thread_data_t *tdata)
     tdata->tid = tid;
 #ifdef BUILD_PT_POST_PROCESSOR
     if (TESTANY(OFFLINE_FILE_TYPE_KERNEL_SYSCALLS, tdata->file_type)) {
+        // We do not support injecting system call trace templates for
+        // raw traces that already have traces for the system calls
+        // collected using Intel-PT.
+        DR_ASSERT(syscall_template_file_ == nullptr);
```

Comment:
This should be a user-facing error I would think.

Reply:
Done.